css/style.css

```diff
@@ -190,6 +190,35 @@ ul {
 
 /* 1st section of the main page */
 
+.hero {
+  background-color: var(--background-color-dark);
+  display: flex;
+  justify-content: center;
+  color: var(--text-color-light);
+  padding: 100px 0;
+}
+.hero-title {
+  font-family: "Roboto-Black", sans-serif;
+  font-size: 2.25rem;
+  margin-bottom: 10px;
+}
```

Comment:
Decide whether you want to always leave one line space between style declarations or have it all one by one. I read on the internet that is better to keep one empty line in style for readability (so I always separate style declarations with one line).